spec/components/publishers/vacancies_component_spec.rb

```diff
@@ -186,4 +187,17 @@
       expect(rendered_component).not_to include(school_cambridge.name)
     end
   end
+
+  describe "heading" do
+    let(:organisation) { create(:school) }
+    let(:vacancy) { create(:vacancy, :published) }
+
+    before { vacancy.organisation_vacancies.create(organisation: organisation) }
+
+    let!(:inline_component) { render_inline(subject) }
```

Comment:
Do we need this line?

Reply:
Yes we do need that line. Could remove the setup for this and move the test for the heading inside the other tests, which are specific to the organisation type though, so would have to replicate it 3 times.